### DISCUSSION TITLE
Rhombic dodecahedron

### DISCUSSION THREAD

**Discussion by KazuhiroOkamura** [neperfepx/neper#672] [Q&A]
The Wigner-Seiz cells for simple cubic, bcc, and fcc lattices are cube, truncated octahedron, and rhombic dodecahedron, respectively. The former two can be tessellated with the properties "cube" and "tocta", respectively. 
Can rhombic dodecahedron be generated?

**Comment by rquey** [comment]
No, rhombic dodecahedra can't be generated. They are degenerate Voronoi tessellation (more than 4 cells meets at a vertex), and so cannot be obtained by the Voronoi tessellation algorithm as truncated octrahedra (which are not degenerate) are. Cubes are also degenerate Voronoi tessellation, but are not generated by the Voronoi tessellation algorithm in Neper.

**Reply by KazuhiroOkamura** [reply]
Thanks for your prompt reply. I understood that it cannot be created due to algorithmic constraints.
